src/test/resources/com/tresys/pcap/pcap.simple.dfdl.xsd

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This simple schema is kindof not very useful. It treats packets as just blobs of data, and requires that each blob is exactly 50 bytes long, which isn't realistic at all. This was originally intended as a way to get some idea of performance with really simple binary data (e.g. no expressions, no choices, no discriminators, etc.). Since no one in the real world should ever use this schema, it really doesn't need to be part of this repo. It's safe to remove.